app/src/main/kotlin/com/hippo/ehviewer/gallery/PageLoader.kt

```diff
@@ -15,14 +15,14 @@ private const val MIN_CACHE_SIZE = 128 * 1024 * 1024
 
 abstract class PageLoader {
     private val cache by lazy {
-        lruCache<Int, Image>(
+        SieveCache<Int, Image>(
```

Comment:
The androidx implementation of SieveCache is not thread-safe, so we need to make sure there's no concurrent modification.

Reply:
Done